.changeset/big-spiders-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hopper-ui/components": patch
```

Comment:
![image](https://github.com/user-attachments/assets/8a3f71f7-bd5f-4dc7-8e31-0c467f7ea41b)
I would showcase both variants on top of each other to avoid issue in moblie.